src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/jpa/VisitNote.kt

```diff
@@ -11,17 +11,23 @@ import javax.persistence.Id
 import javax.persistence.JoinColumn
 import javax.persistence.ManyToOne
 import javax.persistence.Table
+import javax.persistence.UniqueConstraint
 
 @Entity
-@Table(name = "VISIT_NOTES")
+@Table(
+  name = "VISIT_NOTES",
+  uniqueConstraints = [
+    UniqueConstraint(columnNames = ["VISIT_ID", "TYPE"])
+  ]
+)
 data class VisitNote(
 
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   @Column(name = "ID")
   val id: Long = 0,
 
-  @Column(name = "VISIT_ID", unique = true)
+  @Column(name = "VISIT_ID", nullable = false)
```

Comment:
good spot, or perhaps silly me for putting it as unique I think I did actually mean nullable,... Would have thought unit tests would have failed... arr... sql scripts do 
    visit_id           integer          NOT NULL,
Thats why!

Reply:
Shows the tests need some attention